Etg.Yams/Storage/FileMode.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Etg.Yams.Storage
+{
+    public enum FileMode
```

Comment:
Any suggestions? :)

ExistingBinariesAction, ConflictAction, ResolutionMode?

At first, I called it UploadMode but then I reused it for download as well and ended up renaming it to FileMode. Having two separate enums, UploadMode and DownloadMode is also an option, even if they happen to be the same at the moment, they might diverge in the future.

Thoughts?